<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditOrder</class>
 <widget class="QDialog" name="EditOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Order</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/icons/icons/page_edit.png</normaloff>:/icons/icons/page_edit.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="7">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="4" colspan="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="llSerial">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>S&amp;erial:</string>
     </property>
     <property name="buddy">
      <cstring>eSerial</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QComboBox" name="eState">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>201</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>260</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QLineEdit" name="eCost">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QLineEdit" name="eSerial"/>
   </item>
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Order #:</string>
     </property>
     <property name="buddy">
      <cstring>eNumber</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="llDefect">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Claimed defect:</string>
     </property>
     <property name="buddy">
      <cstring>eDisease</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Date:</string>
     </property>
     <property name="buddy">
      <cstring>eDate</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="llCompleteness">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Comp&amp;leteness:</string>
     </property>
     <property name="buddy">
      <cstring>eComplect</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;State:</string>
     </property>
     <property name="buddy">
      <cstring>eState</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="llPhone">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Phone:</string>
     </property>
     <property name="buddy">
      <cstring>ePhone</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDateEdit" name="eDate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="4" colspan="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_14">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Master:</string>
     </property>
     <property name="buddy">
      <cstring>eMaster</cstring>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/door_out.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cPreview">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bPrint">
       <property name="text">
        <string>Print</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/printer.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bSave">
       <property name="text">
        <string>Save and close</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/page_save.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1" colspan="6">
    <widget class="QLineEdit" name="eDisease"/>
   </item>
   <item row="16" column="0" colspan="7">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="6">
    <widget class="QPlainTextEdit" name="eComment">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>81</height>
      </size>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="7">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6" alignment="Qt::AlignLeft">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Customer type</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="rightMargin">
       <number>3</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QRadioButton" name="rbNew">
        <property name="text">
         <string>Ne&amp;w</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QRadioButton" name="rbPrev">
        <property name="text">
         <string>Pre&amp;vious</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbOld">
        <property name="text">
         <string>&amp;Regular</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="cbOldCustomer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="eProductType">
       <property name="minimumSize">
        <size>
         <width>201</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>261</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bAddType">
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Add to catalog</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icons/add.png</normalon>
        </iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0" colspan="7">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="6">
    <widget class="QLineEdit" name="eComplect"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="eCustomer"/>
   </item>
   <item row="10" column="1" colspan="6">
    <widget class="QLineEdit" name="eCond"/>
   </item>
   <item row="15" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_13">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Acceptor:</string>
     </property>
     <property name="buddy">
      <cstring>eAcceptor</cstring>
     </property>
    </widget>
   </item>
   <item row="15" column="6">
    <widget class="QComboBox" name="eMaster">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="llProduct">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Prod&amp;uct:</string>
     </property>
     <property name="buddy">
      <cstring>eProduct</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
    <widget class="QLabel" name="label_12">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>No&amp;tes:</string>
     </property>
     <property name="buddy">
      <cstring>eComment</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="llProductType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Product type:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="llCondition">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cond&amp;ition:</string>
     </property>
     <property name="buddy">
      <cstring>eCond</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="llCustomer">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>eCustomer</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_15">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Approx. cost:</string>
     </property>
     <property name="buddy">
      <cstring>eCost</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLineEdit" name="ePhone"/>
   </item>
   <item row="6" column="6">
    <widget class="QLineEdit" name="eProduct"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="eNumber">
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="3">
    <widget class="QComboBox" name="eAcceptor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="6">
    <widget class="QLineEdit" name="ePrepay">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Prepayment:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>eNumber</tabstop>
  <tabstop>eDate</tabstop>
  <tabstop>eState</tabstop>
  <tabstop>eCustomer</tabstop>
  <tabstop>ePhone</tabstop>
  <tabstop>eProductType</tabstop>
  <tabstop>eProduct</tabstop>
  <tabstop>eSerial</tabstop>
  <tabstop>eDisease</tabstop>
  <tabstop>eCond</tabstop>
  <tabstop>eComplect</tabstop>
  <tabstop>eCost</tabstop>
  <tabstop>eComment</tabstop>
  <tabstop>eAcceptor</tabstop>
  <tabstop>eMaster</tabstop>
  <tabstop>bPrint</tabstop>
  <tabstop>bSave</tabstop>
  <tabstop>bCancel</tabstop>
  <tabstop>cPreview</tabstop>
  <tabstop>bAddType</tabstop>
 </tabstops>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
